src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -249,3 +251,11 @@ function moveFootnotes(document: Document, rootEl: HTMLElement, intl: IntlShape)
     sup.appendChild(link);
   }
 }
+
+function enhanceImagesForAccessibility(rootEl: HTMLElement) {
+  rootEl.querySelectorAll('img').forEach((img) => {
+    img.setAttribute('tabindex', '0');
+    img.setAttribute('role', 'button');
+    img.setAttribute('aria-label', 'Open media preview');
```

Comment:
This will potentially have a bunch of buttons with the same label. If the images reliably have alt text, you can work that into the label (the label will be announced _instead of_ the alt text).

Reply:
Yeah, you're right. It would be confusing. 

I think something like this would be better. 
```
const alt = img.getAttribute('alt');
const label = alt ? `Open preview of ${alt}` : 'Open media preview';
img.setAttribute('aria-label', label);

```